extras/check_unused_names.py

```diff
@@ -96,7 +103,9 @@ def get_names(fname, pattern):
 
 names_unused = names_defined - names_used
 if len(names_unused) != 0:
-    print("Unused Name definition(s): " + ", ".join(names_unused))
+    msg = "unused Name definition(s): " + ", ".join(names_unused)
+    eprint("[NAME] " + msg)
+    print("... {}\\n".format(msg))
```

Comment:
Explain somewhere `eprint()` vs `print()`.

Reply:
I've added a docstring for `eprint()` and the general working of the script in 93258e33d8.